config.test.yaml

```diff
@@ -150,34 +150,144 @@ templates:
-                        uri: http://appservice.wmflabs.org/{domain}/v1/page/mobile-html-sections-lead/{title}
+                        method: get
+                        uri: /{domain}/sys/mobileapps/handling/mobile-html-sections-lead/{title}
+                        headers:
+                          cache-control: '{cache-control}'
             /sections-remaining/{title}:
               get:
+                x-setup-handler:
+                  - init:
+                      method: put
+                      uri: /{domain}/sys/key_rev_value/mobileapps.mobile-html-sections-remaining
+                      body:
+                        revisionRetentionPolicy:
+                          type: 'latest'
+                          count: 1
+                          grace_ttl: 86400
+                        valueType: 'json'
+                        version: 1
                 x-request-handler:
-                  - get_from_backend:
+                  - handle_req:
                       request:
-                        uri: http://appservice.wmflabs.org/{domain}/v1/page/mobile-html-sections-remaining/{title}
+                        method: get
+                        uri: /{domain}/sys/mobileapps/handling/mobile-html-sections-remaining/{title}
+                        headers:
+                          cache-control: '{cache-control}'
             /text/{title}:
               get:
+                x-setup-handler:
+                  - init:
+                      method: put
+                      uri: /{domain}/sys/key_rev_value/mobileapps.mobile-text
+                      body:
+                        revisionRetentionPolicy:
+                          type: 'latest'
+                          count: 1
+                          grace_ttl: 86400
+                        valueType: 'json'
+                        version: 1
                 x-request-handler:
-                  - get_from_backend:
+                  - handle_req:
                       request:
-                        uri: http://appservice.wmflabs.org/{domain}/v1/page/mobile-text/{title}
+                        method: get
+                        uri: /{domain}/sys/mobileapps/handling/mobile-text/{title}
+                        headers:
+                          cache-control: '{cache-control}'
+            /handling/{route}/{title}:
+              get:
+                x-request-handler:
+                  - get_rev:
+                      request:
+                        method: get
+                        uri: /{domain}/sys/page_revisions/page/{title}
+                        headers:
+                          cache-control: '{cache-control}'
+                  - check_storage:
+                      request:
+                        method: get
+                        uri: /{domain}/sys/key_rev_value/mobileapps.{$.request.params.route}/{$.request.params.title}/{$.get_rev.body.items[0].rev}/{$.get_rev.body.items[0].tid}
+                      return_if:
```

Comment:
My be better to use short notation for `{route}` and `{title}` to avoid a super-long line?